test/integration/response-test.js

```diff
@@ -8,6 +8,14 @@ const expect = require('chai').expect;
 describe('Response', () => {
   afterEach(() => mockyeah.reset());
 
+  it('should not allow for more than one response type', () => {
+    const responsePropTypes = ['fixture', 'filePath', 'html', 'json', 'text'];
+
+    expect(() => {
+      mockyeah.get('/service/end/point', { fixture: './test/fixtures/some-data.json', filePath: './test/fixtures/some-data.csv' });
+    }).to.throw('Response options must not include more than one of the following: ' + responsePropTypes.join(', '));
+  });
+
```

Comment:
There could be more coverage here... it would be good to cover all type combinations. Iterating through them might be a good solve. Thoughts? Also, `responsePropTypes ` could be updated to match the language in `server/lib/RouteResolver.js`.